datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/dashboards.py

```diff
@@ -53,11 +53,17 @@ def dashboards():
                 display_queue.append(
                     (
                         echo_failure,
-                        '    {} is using the new /dash payload format which isn\'t currently supported.'
-                        ' Please use the format from the /screen or /time API endpoints instead.'.format(
-                            dashboard_file
-                        ),
-                    )
+                        f'    {dashboard_file} is using the new /dash payload format which isn\'t currently supported.'
+                        ' Please use the format from the /screen or /time API endpoints instead.',
+                    ),
+                )
+
+            all_keys = set(decoded.keys())
+            if not REQUIRED_ATTRIBUTES.issubset(all_keys):
+                missing_fields = REQUIRED_ATTRIBUTES.difference(all_keys)
+                file_failed = True
+                display_queue.append(
+                    (echo_failure, f"    {dashboard_file} does not contain the required fields: {missing_fields}"),
```

Comment:
nit: not sure why the fields is showing up under brackets, could we use a list like 
```
...required fields: ['abc', 'efg']
```

currently shows up in a bracket
```
    /Users/christine.chen/dev/integrations-core/airflow/assets/dashboards/overview.json does not contain the required fields: {'board_title'}
```